### ISSUE TITLE
maxItems setting not working correctly

### ISSUE THREAD

**Issue by chipcop106** [lukasoppermann/html5sortable#844]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to ['Homepage demo'](http://lukasoppermann.github.io/html5sortable/index.html)
2. Scroll down to "Connected lists"
3. You can see in the white list nested child limit 3 but I can drag 4 items inside the list. I have the same bug in my site, I set option maxItems = 1 but when I test I can drop a maximum of 2 items inside the list.

**Expected behaviour**
This featured working correctly

![image](https://user-images.githubusercontent.com/17249557/142558138-ebdacb02-e74f-4baa-815a-80702f7bce3c.png)

**Comment by lukasoppermann** [comment]
Hey, sorry for the late reply.

I assume this is a simplebug.
Instead of using:
```js
data(sortableElement, 'items')).length > parseInt(options.maxItems)
```
in line 639 it would have to be:
```js
data(sortableElement, 'items')).length >= parseInt(options.maxItems)
```
https://github.com/lukasoppermann/html5sortable/blob/master/src/html5sortable.ts#L639-L641

Would you be able to change it, test it and if this fixes it send a PR? That would be awesome.

**Comment by norbitrial** [comment]
> https://github.com/lukasoppermann/html5sortable/blob/master/src/html5sortable.ts#L639-L641
> 
> Would you be able to change it, test it and if this fixes it send a PR? That would be awesome.

@lukasoppermann Hey! 👋 Not sure if still help needed here, I've opened the above PR for fixing this issue. I've tested on my local and your suggestion seems to be solving the issue. 🚀 

See the details of my tests in #862. Let me know if anything else should be included, I've intentionally not committed the **dist** folder as mentioned in the [contributing guideline](https://github.com/lukasoppermann/html5sortable/blob/6894f4727f86832cea08370c6d4167a225c0d115/CONTRIBUTING.md#pull-requests).
